src/aux-server/aux-web/aux-player/PlayerHome/PlayerHome.vue

```diff
@@ -70,6 +71,12 @@
                     </md-select>
                 </md-field>
 
+                <md-field v-if="showInstNameInput">
+                    <label for="instName">inst name</label>
+                    <md-input name="instName" id="instName" v-model="instName" />
+                    <field-errors field="instName" :errors="errors" />
+                </md-field>
```

Comment:
@TroyceGowdy This looks pretty good. One UX improvement I can think of is to put the automatically generated inst name in the input as the placeholder so the user can see what the new inst will be called if they don't enter a custom name.